packages/itwinui-react/src/core/Toast/Toast.tsx

```diff
@@ -35,6 +36,18 @@ export type ToastProps = {
    * Content of the Toast message
    */
   content: React.ReactNode;
+  /**
+   * Passes props to content
+   */
+  contentProps?: React.ComponentPropsWithoutRef<'div'>;
+  /**
+   * Passes props to status area
+   */
+  statusAreaProps?: React.ComponentPropsWithoutRef<'div'>;
```

Comment:
Not sure if needed?
I can't imagine a scenario where user want to customise this area.

Reply:
i'm ok with leaving this one out for now. Most important parts are the toast and the toast-message

Reply:
Both options sound good to me. Since it is already added, might be better to leave it.

Reply:
> Since it is already added, might be better to leave it.

It takes 15 seconds to remove it, so this is not a strong argument. It's always worth taking the extra time to think about the public-facing API; we don't want to carelessly add undesired props just because we didn't want to spend a few extra seconds.

Reply:
> It takes 15 seconds to remove it, so this is not a strong argument. 

I don't mean to save time by doing things hastily. I just meant to say if we are all on the fence about whether to keep it or not, no need to bother removing it. But if we all feel strong to remove it, then let's remove it.

> It's always worth taking the extra time to think about the public-facing API; we don't want to carelessly add undesired props just because we didn't want to spend a few extra seconds.

For sure, I definitely agree. But I think that in general, if exposing one prop would help some user in the future, it may not be wise to be this conservative on our end. But if the cons of including the prop outweigh the added customizability, let's not expose it.

In this case, I am okay with both: keeping or removing `statusAreaProps`.

Reply:
Removed. It's easier to add props in a future than to remove or change them.